test/index.test.js

```diff
@@ -89,22 +89,21 @@ describe('KISSmetrics', function() {
         window.KM_SKIP_PAGE_VIEW = 1;
       });
 
-      it('should record normal kissmetrics page views when the option is set', function() {
-        window.KM_SKIP_PAGE_VIEW = false;
-        analytics.page();
-        analytics.didNotCall(window._kmq.push);
-      });
-
       it('should call `KM.pageView()` when KM_SKIP_PAGE_VIEW is not set', function() {
         window.KM_SKIP_PAGE_VIEW = false;
         analytics.page();
-        analytics.calledOnce(window.KM.pageView);
+        analytics.called(window._kmq.push, ['record', 'Page View', {
+          'Viewed URL': window.location.href,
+          Referrer: document.referrer || 'Direct'
```

Comment:
@chriso I do not think you can set `document.referrer = ...` because it is a read only property (I vaguely remember running into this)

I think for this test, you could do something like the following since you want to test that manually passed in page properties are mapped:

```js
var referrer = 'www.yolo.com';
var url = 'www.yolo.com/now';
analytics.page({
  referrer: referrer,
  url: url
});

analytics.called(window._kmq.push, ['record', 'Page View', {
   'Viewed URL': url,
   Referrer: referrer || 'Direct'
}]);